front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -858,7 +860,7 @@ export function retrievalMetaPromptMutiActions() {
 let REFS: string[] | null = null;
 const getRand = rand("chawarma");
 
-const getRefs = () => {
+export const getRefs = () => {
```

Comment:
I think we want to move that in lib/api/assistant/actions/citations.ts along with the potentially shared meta prompt?